modules/media/ajax/FileUpload.php

```diff
@@ -347,13 +349,12 @@ function toSelect($options, $item, $item2)
 {
     $selectOptions = [];
 
-    $optionsValue = $item;
-    if (isset($item2)) {
-        $optionsValue = $item2;
-    }
+    $optionsVal = $item2 ?? $item;
```

Comment:
@um4r12 please update the php version at the top of the file to 7

Reply:
@ridz1208 done.